android/app/src/e2e/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config xmlns:tools="http://schemas.android.com/tools">
+	<base-config cleartextTrafficPermitted="false">
+		<trust-anchors>
+			<certificates src="system" />
+			<certificates src="user"
+				tools:ignore="AcceptsUserCertificates" />
+		</trust-anchors>
+	</base-config>
```

Comment:
Do we need this?

Reply:
Just to disable clearText communication on remote hosts. Making as little changes to the execution as possible. Would give more consistent results for remote servers in case of certificate/network issues.